test/email_test.exs

```diff
@@ -0,0 +1,74 @@
+defmodule Fields.EmailTest do
+  use ExUnit.Case
+  alias Fields.{EmailPlaintext, EmailHash, EmailEncrypted}
+
+  describe "types" do
+    test "EmailPlaintext.type is :string" do
+      assert EmailPlaintext.type() == :string
+    end
+
+    test "EmailEncrypted.type is :binary" do
+      assert EmailEncrypted.type() == :binary
+    end
+
+    test "EmailHash.type is :binary" do
+      assert EmailHash.type() == :binary
+    end
+  end
+
+  describe "cast" do
+    test "Email.cast accepts a string" do
+      assert {:ok, "test@test.com"} == EmailPlaintext.cast("test@test.com")
```

Comment:
do we have a situation where storing an email in `plaintext` is desirable?

Reply:
Yeah that's what I was thinking, for publicly available email addresses, such as those of the venues in CS, it probably wouldn't be necessary to encrypt them